content/html/concepts/entities/entities.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Entities'
+Description: 'Entities are character sequences placed in the text of an HTML document to print out reserved or other special characters.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Characters'
+  - 'Symbol'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Entities are character sequences placed in the text of an HTML document to print out reserved or other special characters. When parsing the HTML, a browser replaces the entity's sequence of characters with the character it represents.
```

Comment:
including a little more on why entities would be used
```suggestion
Entities are character sequences placed in the text of an HTML document to print out reserved or other special characters which may otherwise be interpreted as HTML code by a browser. When parsing the HTML, a browser replaces the entity's character sequence with the character it represents.
```